lib/frontend/github-oauth.js

```diff
@@ -7,8 +7,10 @@
 const request = require('request');
 const crypto = require('crypto');
 const url = require('url');
+const { Installation } = require('../models');
+const { AppSecrets } = require('../models');
 
-const host = process.env.GHE_HOST || 'github.com';
+let host = 'github.com';
```

Comment:
is this not required process.env.GHE_HOST?

Reply:
No. GHE_HOST is used when you want to make jira app to work with Github Enterprise server. https://github.com/probot/probot/blob/7a43e9e5d41cbe71ca416b3780c93a6c303c739a/docs/github-api.md#github-enterprise. 

Right now I have separated out UI for github.com and ghae. for ghae user can input host url. else by default github.com will be used.